RealmSwift/Migration.swift

```diff
@@ -80,7 +80,7 @@ extension Realm {
  instance provides access to the old and new database schemas, the objects in the Realm, and provides functionality for
  modifying the Realm during the migration.
  */
-public final class Migration {
+public struct Migration {
```

Comment:
I assume that changing from class to struct doesn't in any way change how users interact with these types (i.e., it's not a breaking change)?